radio/src/Makefile

```diff
@@ -52,7 +52,7 @@ DELAY = 5
 # PCB version
 # Values: 9X, 9X128, 9X2561, 9XR, 9XR128, 9XR2561, GRUVIN9X, MEGA2560, SKY9X, 9XRPRO, AR9X, TARANIS
 # Old values STD and STD128 are mapped to 9X and 9X128 for compatibility
-PCB = 9X
+PCB = 9XRPRO
```

Comment:
It would have worked that way if you had created your branch *before* commtting your makefile changes on master (or alternatively rebased it to a point of master before these changes were committed after the fact)